docker/nginx/conf.d/server/server.api

```diff
@@ -228,12 +227,26 @@ location /skynet/skyfile {
 
     # proxy this call to siad endpoint (make sure the ip is correct)
     proxy_pass http://sia:9980/skynet/skyfile/$dir1/$dir2/$dir3$is_args$args;
+
+    log_by_lua_block {
```

Comment:
worthwhile to dedupe this block through an include?

Reply:
I deliberately squashed the include file because it hides the functionality and the lua block used. It's easier to maintain when I know exactly what blocks are used on a route. Ideally we would have no include files and move most of the business logic to separate lua files that can be unit tested. The idea is that if there is a functionality that can be encapsulated in include file, it should be possible to also encapsulate it in a testable lua function.